sustainability/src/main/webapp/WEB-INF/views/admin/concepts/configuration.jsp

```diff
@@ -0,0 +1,31 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="sec"
+	uri="http://www.springframework.org/security/tags"%>
+
+<!-- Custom styling for configuration page. -->
+<link href="<c:url value="/resources/bootstrap/css/admin/admin.css" />" rel="stylesheet">
+
+<div class="panel panel-default">
+  <c:forEach items="${concepts}" var="concept">
+    <div class="panel-heading">
+      <p><strong>${concept.name}</strong></p>
+    </div>
+    
+    <div>
+      <c:forEach items="${concept.children}" var="child">
+        <div class="panel-body"">
+	      <li class="list-group-item">
+	        ${child.name}
+	        <c:forEach items="${roles}" var="role">
+	          <label class="switch checkbox-switch">
+	            <input type="checkbox" class="default" name="${role}" value="${role}"> ${role}
```

Comment:
name and value should be different (value can change over time as stakeholders change their mind, values need to stay the same in most cases for backwards compatibility).